src/scss/components/_threeImages.scss

```diff
@@ -0,0 +1,45 @@
+.images{
+    margin: 60px 0;
+
+    &__item{
+        width: 80%;
+        height: 168px;
+
+        img{
+            object-fit: cover;
+            width: 100%;
+            height: 100%;
+        }
+
+        &:nth-child(odd){ 
+            margin-bottom: 16px;
+        }
+
+        &:nth-child(even){
+            margin-bottom: 24px;
+            margin-left: auto;
+        }
+
+    }    
+
+    @include breakpoint(md){
+        margin: auto;
+        margin-top: 168px;
+        margin-bottom: 200px;
+
+        &__item{
+            height: 297px;
```

Comment:
Same here. Don't set a fixed height. Let the image height to be automatic. You can remove this.